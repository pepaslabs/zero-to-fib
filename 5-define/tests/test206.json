../../4-lambda/tests/test206.json