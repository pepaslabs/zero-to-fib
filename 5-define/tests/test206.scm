../../4-lambda/tests/test206.scm