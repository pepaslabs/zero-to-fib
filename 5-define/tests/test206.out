../../4-lambda/tests/test206.out